invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentMetadata/DocumentMetadata.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import {
+  Grid,
+  Segment,
+  Container,
+  Header,
+  Button,
+  Icon,
+} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { MetadataTable } from '../../../components/MetadataTable';
+
+export default class DocumentMetadata extends Component {
+  openEditor(url) {
+    window.open(`/editor?url=${url}`, url);
+  }
+
+  render() {
+    const data = {
+      ...this.props.documentDetails,
+      ...this.props.documentDetails.metadata,
+    };
+
+    const documentUrl = this.props.documentDetails.links.self;
+
+    const rows = [
+      { name: 'Created', value: data.created },
+      { name: 'Updated', value: data.updated },
+      { name: 'Title', value: data.title },
```

Comment:
Are these mandatory fields in the schema?

Reply:
yes (in the final state of the schema), but in general they should be checked, also in the rest of the components